Stripe/PublicHeaders/STPApplePay.h

```diff
@@ -1,24 +1,24 @@
 //
-//  STPApplePayPaymentMethod.h
+//  STPApplePay.h
 //  Stripe
 //
 //  Created by Ben Guo on 4/19/16.
 //  Copyright © 2016 Stripe, Inc. All rights reserved.
 //
 
 #import <Foundation/Foundation.h>
-#import "STPPaymentMethod.h"
+#import "STPPaymentOption.h"
 
 /**
  An empty class representing that the user wishes to pay via Apple Pay. This can
  be checked on an `STPPaymentContext`, e.g:
  
  ```
- if ([paymentContext.selectedPaymentMethod isKindOfClass:[STPApplePayPaymentMethod class]]) {
+ if ([paymentContext.selectedPaymentOption isKindOfClass:[STPApplePay class]]) {
     // Don't ask the user for their card number; they want to pay with apple pay.
  }
  ```
  */
-@interface STPApplePayPaymentMethod : NSObject <STPPaymentMethod>
+@interface STPApplePay : NSObject <STPPaymentOption>
```

Comment:
Should this be `STPApplePayPaymentOption`?

Reply:
Only thought with `STPApplePay` is that the name might be more consistent with the other options - `STPCard`, `STPSource`.  

I don't have a strong opinion either way, LMK what you prefer

Reply:
I don't have a strong preference either way either, but since `STPApplePay` isn't backed by a Stripe API object, I'd lean towards being more specific and calling it `STPApplePayPaymentOption`